modules/alloy/src/main/ts/ephox/alloy/aria/AriaOwner.ts

```diff
@@ -1,46 +1,7 @@
-import { Id, Optional } from '@ephox/katamari';
-import { Attribute, PredicateFind, SelectorFind, SugarElement, SugarNode, SugarShadowDom } from '@ephox/sugar';
+import * as AriaManager from './AriaManager';
 
-const find = (queryElem: SugarElement<Node>): Optional<SugarElement<Element>> => {
-  const dependent = PredicateFind.closest(queryElem, (elem): elem is SugarElement<Element> => {
-    if (!SugarNode.isElement(elem)) {
-      return false;
-    }
-    const id = Attribute.get(elem, 'id');
-    return id !== undefined && id.indexOf('aria-owns') > -1;
-  });
-
-  return dependent.bind((dep) => {
-    const id = Attribute.get(dep, 'id');
-    const dos = SugarShadowDom.getRootNode(dep);
-
-    return SelectorFind.descendant(dos, '[aria-owns="' + id + '"]');
-  });
-};
-
-export interface AriaManager {
-  id: string;
-  link: (elem: SugarElement<Element>) => void;
-  unlink: (elem: SugarElement<Element>) => void;
-}
-
-const manager = (): AriaManager => {
-  const ariaId = Id.generate('aria-owns');
-
-  const link = (elem: SugarElement<Element>) => {
-    Attribute.set(elem, 'aria-owns', ariaId);
-  };
-
-  const unlink = (elem: SugarElement<Element>) => {
-    Attribute.remove(elem, 'aria-owns');
-  };
-
-  return {
-    id: ariaId,
-    link,
-    unlink
-  };
-};
+const find = AriaManager.find(AriaManager.LinkableAttribute.AriaOwns);
```

Comment:
Minor. I wonder if we should just combine AriaOwner and AriaControls now. I guess it's unclear what to call the combination.

Reply:
It's all the one thing now anyways, as we didn't need the `AriaOwner` module in the end (see my other comment).